specification/electronic-prescription-service-api.yaml

```diff
@@ -339,9 +344,9 @@ paths:
         ## Overview
         Use this endpoint to send a prescription release request to EPS.
 
-        This will gather a set of prescriptions based on the information passed to the endpoint: 
-        * to gather prescription information for an entire pharmacy, send owner information
-        * to gather prescription information for a specific prescription, send the group identifier for that prescription
+        This will return a set of prescriptions based on the information passed to the endpoint: 
```

Comment:
This returns a set of prescriptions based on the information passed to the endpoint: